starter_code/views/partials/beerPartial.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="bloque">
+     <img src="{this.image_url}">
+          <h5 class="card-title">{{this.name}}</h5>
```

Comment:
Dentro del src de la imagen también se ponen dobles llaves
<img src="{{this.image_url}}">